test/test_code_to_gast_conditionals.py

```diff
@@ -1,19 +1,20 @@
-import sys
 import unittest2
-
+import sys
+ 
 sys.path.append('code_to_gast')
 sys.path.append('code_to_gast/assign')
 sys.path.append('code_to_gast/expression')
 sys.path.append('code_to_gast/conditional')
 sys.path.append('code_to_gast/helpers')
 sys.path.append('code_to_gast/routers')
 
+import py_main
 import js_main
 
 
-class test_js_to_gast_conditionals(unittest2.TestCase):
+class test_code_to_gast_conditionals(unittest2.TestCase):
     maxDiff = None
-    def test_ifs(self):
+    def test_ifs_js(self):
```

Comment:
maybe should each test for js and py be combined into single ones? Meaning just have test_ifs and then include both the python and javascript in the single test? That way each expected_gast does not have to be repeated

Reply:
Yeah that is a good idea. I did that in my other file but didn't think about it here.